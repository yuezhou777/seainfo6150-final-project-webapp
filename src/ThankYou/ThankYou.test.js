import React from "react";
import { render } from "@testing-library/react";
import Thankyou from "./Thankyou.jsx";
import { BrowserRouter } from 'react-router-dom';

it("renders Thankyou Correctly", () => {

    const { container } = render(
        <BrowserRouter>
            <Thankyou />
        </BrowserRouter>
    );

    expect(container).toMatchSnapshot();
});