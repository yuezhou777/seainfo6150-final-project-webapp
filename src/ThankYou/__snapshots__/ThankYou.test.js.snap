// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Thankyou Correctly 1`] = `
<div>
  <div>
    <div>
      <div
        class="navContainer"
      >
        <header
          class="navContainer"
        >
          <nav>
            <a
              class="navIcon"
              href="/"
            >
              PurrPet
            </a>
          </nav>
          <nav>
            <ul
              class="navList"
            >
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/aboutus"
                >
                  About Us
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/allpets"
                >
                  All Pets
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/category"
                >
                  Categories
                </a>
              </li>
              <li
                class="navItem"
              >
                <a
                  class="link"
                  href="/contactus"
                >
                  Contact Us
                </a>
              </li>
            </ul>
          </nav>
        </header>
      </div>
    </div>
    <div
      class="container"
    >
      <h2
        class="contactTitle"
      >
        Thank you for adopting!
      </h2>
      <div
        class="contactContent"
      >
        <h3>
          We will contact you soon about the following process.
        </h3>
      </div>
      <div>
        <img
          alt="Thank you image: a smiling dog"
          src="thank-you.jpeg"
        />
      </div>
      <div>
        <a
          class="link"
          href="/"
        >
          Back To Main
        </a>
      </div>
    </div>
  </div>
</div>
`;
